puppet/modules/jenkins_node/manifests/init.pp

```diff
@@ -23,12 +23,19 @@
   include fastly_purge
 
   if $facts['os']['family'] == 'RedHat' {
-    $java_package = 'java-11-openjdk-headless'
+    $java_package = 'java-17-openjdk-headless'
 
-    package { ['java-1.8.0-openjdk', 'java-1.8.0-openjdk-headless', 'java-1.8.0-openjdk-devel']:
+    package { ['java-11-openjdk', 'java-11-openjdk-headless', 'java-11-openjdk-devel']:
```

Comment:
I was wondering if we should have `jenkins_java_version = 17` and `jenkins_java_cleanup = 11` somewhere, and just reuse that through the manifests.